package com.lianshengtong.lsc.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;
import java.math.BigDecimal;
import java.time.LocalDateTime;

@Data
@TableName("product")
public class Product {
    @TableId(type = IdType.AUTO)
    private Long id;
    private Long merchantId;
    private String productName;
    private String productDesc;
    private BigDecimal price;
    private Long lscPrice;
    private Integer stock;
    private Integer sales;
    private String productImages;
    private Integer aiReviewResult;
    private String aiTags;
    private Integer status;
    private LocalDateTime createdAt;
}
